### DISCUSSION TITLE
Sort by most recent

### DISCUSSION THREAD

**Discussion by Gessink** [music-assistant/support#579] [Feature requests and ideas]
Natively spotify sorts most of it's list based on "Most recent" I think this is for different use cases either one of "Most recently played" or "Most recently added". Would it be easy to add both or one of these options to music assistant as well? 

This would make lists look a lot more recognizable for me and probably a lot of people

**Comment by marcelveldt** [comment]
The lists sorting options are now extended with "recently added" in MA 2022.7 

Most recently played will be added later

**Reply by mikaeljagelid** [reply]
Well, to be useful the sorting options will need to support switching between ascending and descending.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
